dnase/run_delly.bash

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
hmm, did you mean to commit this?

Reply:
I added all files by mistake. I will remove this file.